docs/mccp/usage-guide.md

```diff
@@ -24,92 +26,84 @@ node/ip-192-168-68-212.eu-north-1.compute.internal
 $ kubectl label node/ip-192-168-40-197.eu-north-1.compute.internal wkp-database-volume-node=true
 ```
 
-Now you can enable the MCCP in config.yaml.
+This label will ensure that the MCCP pods will be deployed and run on the selected node, if the **SQLite** option has been chosen.
 
 :::info
-_The MCCP also supports **PostgreSQL** or mounting SQLite on Persistent Volume instead of the host volume described here. See [Database Configuration](./database-configuration) for details._
+_MCCP also supports **PostgreSQL** or mounting SQLite on Persistent Volume instead of the host volume described here. See [Database Configuration](./database-configuration) for details._
 :::
 
-### 2. Enable the MCCP in config.yaml
+### 2. Creating a secret for docker repository
```

Comment:
Should we add _Create a namespace_

e.g. 

Run `kubectl create namespace mccp` somewhere too? 🤔

Reply:
Added step for creating a namespace 👍